test_integration/appIntegrationTest.js

```diff
@@ -29,61 +33,166 @@ var updatedAppConfig = {
   maintenance: true,
   stack: 'cedar-14',
   config_vars: {
-    NODE_ENV: 'development',
+    NODE_ENV: 'production',
     ANOTHER_FEATURE_FLAG: 'true'
   },
-  addons: {librato: {plan: 'librato:development'}},
+  addons: {
+    librato: {plan: 'librato:development', name: 'updated-app-librato'},
+    logentries: {plan: 'logentries:le_tryit', name: 'updated-logentries'},
+    'heroku-redis': {plan: 'heroku-redis:hobby-dev', name: 'heroin-sample-redis'}
+  },
   collaborators: ['mateusz.kwasniewski@schibsted.pl'],
   features: {
     'log-runtime-metrics': {enabled: false},
   },
   formation: [{process: 'web', quantity: 1, size: 'Free'}]
 };
 
+var anotherAppConfig = {
```

Comment:
@pmrukot I'm wondering if moving the addon related stuff to a separate addonIntegrationTest.js would show the test intent better? What do you think?